1-trigger-TeamCity.md

```diff
@@ -0,0 +1,56 @@
+# Triggering a Teamcity build from a push GitHub
```

Comment:
Is this document necessary? I think we already assume the readers know how to do this step.

Reply:
i think it is no problem